CHANGELOG.md

```diff
@@ -7,10 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Added
 
 ### Changed
+- Use `https://` protocol instead of `git://` to clone postgres pldebugger repository in flowdb Dockerfile
```

Comment:
Maybe make the changelog entry for this a fixed one instead - the intent of this is to fix being unable to build if the port git:// isn't open, right?

Reply:
Yup, correct.